cli/data.go

```diff
@@ -565,6 +623,74 @@ func (c *viamClient) deleteBinaryData(filter *datapb.Filter) error {
 	return nil
 }
 
+func (c *viamClient) addTagsToBinary(filter *datapb.Filter, tags []string) error {
+	if err := c.ensureLoggedIn(); err != nil {
+		return err
+	}
+	_, err := c.dataClient.AddTagsToBinaryDataByFilter(context.Background(),
+		&datapb.AddTagsToBinaryDataByFilterRequest{Filter: filter, Tags: tags})
+	if err != nil {
+		return errors.Wrapf(err, "received error from server")
+	}
+	print(c.c.App.Writer, "Successfully tagged data")
+	return nil
+
+}
+func (c *viamClient) removeTagsFromBinary(filter *datapb.Filter, tags []string) error {
+	if err := c.ensureLoggedIn(); err != nil {
+		return err
+	}
+	_, err := c.dataClient.RemoveTagsFromBinaryDataByFilter(context.Background(),
+		&datapb.RemoveTagsFromBinaryDataByFilterRequest{Filter: filter, Tags: tags})
+	if err != nil {
+		return errors.Wrapf(err, "received error from server")
+	}
+	print(c.c.App.Writer, "Successfully removed tags from data")
+	return nil
+}
+
+func (c *viamClient) dataAddTagsToDataByIDs(tags []string, orgID, locationID string, fileIDs []string) error {
+	if err := c.ensureLoggedIn(); err != nil {
+		return err
+	}
+	binaryData := make([]*datapb.BinaryID, 0, len(fileIDs))
+	for _, fileID := range fileIDs {
+		binaryData = append(binaryData, &datapb.BinaryID{
+			OrganizationId: orgID,
+			LocationId:     locationID,
+			FileId:         fileID,
+		})
+	}
+	_, err := c.dataClient.AddTagsToBinaryDataByIDs(context.Background(),
+		&datapb.AddTagsToBinaryDataByIDsRequest{Tags: tags, BinaryIds: binaryData})
+	if err != nil {
+		return errors.Wrapf(err, "received error from server")
+	}
+	printf(c.c.App.Writer, "Added tags %v to data", tags)
+	return nil
+}
+
+func (c *viamClient) dataRemoveTagsFromData(tags []string, orgID, locationID string, fileIDs []string) error {
```

Comment:
For consistency with the other ByIds method, maybe `dataRemoveTagsFromDataByIDs`